tests/integration_tests/test_examples.py

```diff
@@ -24,7 +24,8 @@
     EXAMPLES_ROOT_DIR / 'torch/maml_trpo_half_cheetah_dir.py',
     EXAMPLES_ROOT_DIR / 'torch/maml_vpg_half_cheetah_dir.py',
     EXAMPLES_ROOT_DIR / 'torch/maml_trpo_ml10.py',
-    EXAMPLES_ROOT_DIR / 'torch/pearl_ml1_push.py'
+    EXAMPLES_ROOT_DIR / 'torch/pearl_half_cheetah_vel.py',
```

Comment:
hmm is there a particular reason these are long-running? do they take a long time to setup?

usually example tests should run only for 1 epoch (the actual example can run for longer).

@gitanshu can help here

Reply:
Also if you're adding a test here because it takes too long to run, you'd have to add new test for it

Reply:
GPU isn't available so these tests are taking a long time to run.

Reply:
does it take a huge time to run 1 iteration or many?